test/fixtures/pullrequests/pullrequests.js

```diff
@@ -0,0 +1,41 @@
+const githubPRInfo = require('../contributions/githubPRInfo')()
+
+/**
+ * Mocking response from GitHub APIs for open pull requests
+ * These are open PRs in descending order (newest first)
+ *
+ * @return {Object}
+ */
+const getOpenPRs = () => {
+  const allPRs = githubPRInfo.prakash.data.items
```

Comment:
Can we customise this to explicitly pass the data items and get PRs from it?

We might also need to add another object further for a different use case in fixtures.

Reply:
I get what you mean but it's highly unlikely that we'll be modifying the hierarchy of the structure since that is exactly what is provided by GitHub API. A big object >> `data` >> `items` contains our PRs.

Reply:
I mean, we might need to add another github response around PRs for some another use case. Refer the users' fixtures.
Thus, we should be able to pass the custom object to this function to get its mutated response. Currently, its giving only the response created from `githubPRInfo.prakash.data.items` and is not customisable.

Reply:
The `githubPRInfo` file has data for more use cases like user with no PR... The new fixtures might end looking really close to the already existing `githubPRinfo`, can you please check that once?

Reply:
Yup, it will look the same, but we won't be able to get that data from `getOpenPRs()` function, as its only giving back the data from  `githubPRInfo.prakash.data.items`. If to morrow I added a new response item in that file, I wont get that with `getOpenPRs()` as it does not accept any custom arguments.